inspire/modules/migrator/__init__.py

```diff
@@ -0,0 +1 @@
+from .receivers import *
```

Comment:
It's kind of a pattern we have been using in some other modules in order to connect the receivers to the signals.